server/utils.go

```diff
@@ -105,9 +105,11 @@ func writeAuthzResponse(w http.ResponseWriter, spec *authzv1beta1.SubjectAccessR
 	}
 
 	w.WriteHeader(http.StatusOK)
-	if glog.V(10) {
-		data, _ := json.MarshalIndent(resp, "", "  ")
-		glog.V(10).Infof("final data:%s", string(data))
+	if glog.V(7) {
+		if _, ok := spec.Extra["oid"]; ok {
+			data, _ := json.MarshalIndent(resp, "", "  ")
```

Comment:
we should refrain from MarshalIndent

Reply:
any specific reason? We are doing this to get the logs in the pretty format hence asking

Reply:
multiple lines are easy to read but horrible at log processing like kusto

Reply:
removed